frontend/lib/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.tsx

```diff
@@ -470,7 +499,6 @@ export function RenderedMarkdown({
                 userSelect: "none",
                 verticalAlign: "bottom",
                 whiteSpace: "nowrap",
-                wordWrap: "normal",
```

Comment:
whats the reason why this needs to be removed?

Reply:
Oh sorry, I removed that previously when I was playing around with enabling rainbow colors, but forgot to add it back in.